app/main.py

```diff
@@ -1,6 +1,30 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("No API_KEY found in environment variables")
+
+    city = "Paris"
+    url = f"https://api.weatherapi.com/v1/current.json?key={api_key}&q={city}"
```

Comment:
The variable 'url' should be in uppercase since it is a constant. This aligns with the checklist requirement: 'Use uppercase style for constants' names'.